features/support/timecop.rb

```diff
@@ -6,7 +6,16 @@ class Timecop # rubocop:todo Style/Documentation
   class << self
     # Block the use of Timecop.freeze as it upsets the Capybara...
     def freeze_with_warning(_time)
-      raise "\n\n#{'*' * 90}\nTimecop.freeze() interferes with Capybara's javascript timeouts.\nCould you either use Timecop.travel instead or not use JavaScript in this scenario?  \n\n#{'*' * 90}\n\n"
+      raise <<~EXCEPTION
+
+        #{'*' * 90}
```

Comment:
I think you might be missing a new line character at the start and the end of this heredoc.

Reply:
You're right. This was half intentional, as adding two seemed absurd, however when actually printed to the console this would just result in a single blank line, not the two that the heredoc shows, so I'll add the extra new line in.